### PR TITLE
feat(router): add Router.re for handle router in web (client side)

### PR DESCRIPTION
**Pull request by r17x**

### CODE REVIEW

File: web/src/Router.re [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  | ["/"] => Home
+  | ["/new"] => New
+  | ["/stats", slug] => Stats(slug)
+  | _ => Notfound
+  };
+
+/*
+  use for handle button or ahref click
+ */
+let linkTo = (path, event) => {
```
Comment (faultables) [inline web/src/Router.re:25]: I don't see this used anywhere?

Comment (r17x) [reply web/src/Router.re:25]: example usage of `linkTo`. 

``` 
<button onClick={event => Home->Router.unwrap->Router.linkTo(event)}  />
```

**Comment by faultables** [comment]
LGTM

File: web/src/Router.re (hunk 1 above)
Comment (faultables) [reply web/src/Router.re:25]: any conflict with this?

```suggestion
let make = (~href, ~className="", ~style: ReactDOMRe.Style.t=?, ~children) => {
  let handleClick = (href, e) => {
    ReactEvent.Mouse.preventDefault(e);
    ReasonReactRouter.push("/#" ++ href);
  };

  <a href={"/#" ++ href} style className onClick={e => handleClick(href, e)}>
    children
  </a>;
};
```

https://github.com/evilfactorylabs/gow/blob/master/web/src/components/Link.re

Comment (r17x) [reply web/src/Router.re:25]: I think not conflict, as `linkTo` just functional and `Link` is component based :D

Comment (faultables) [reply web/src/Router.re:25]: OK thanks, let's see where we can use `linkTo`method. thank you!

**Review by faultables — APPROVED** [review]
L G T M

**Pull request merged**